internal/server/models/access_key.go

```diff
@@ -31,6 +30,8 @@ type AccessKey struct {
 	KeyID          string `gorm:"<-;uniqueIndex:idx_access_keys_key_id,where:deleted_at is NULL"`
 	Secret         string `gorm:"-"`
 	SecretChecksum []byte
+
+	Scopes CommaSeparatedStrings // if set, scopes limit what the key can be used for
```

Comment:
Any reason this was moved? Not a big deal in any case.

Reply:
Good question! Yes these fields on models moved so that the order would match the order that migrations were applied. Otherwise the initial schema does not match the migrated schema because the fields are in different orders.

I had to change a few of the fields in models to move newly added fields to the end of the struct.